tests/doc/stringify.ts

```diff
@@ -330,6 +330,36 @@ z:
     expect(String(doc)).toBe('? >+ #comment\n  foo\n\n: bar\n')
   })
 
+  test('Keep pipe when empty block literal is given', () => {
+    const doc = YAML.parseDocument('|-')
+    expect(String(doc)).toBe('|\n\n')
+  })
+
+  test('Keep pipe when pipe is provided', () => {
+    const doc = YAML.parseDocument('|')
+    expect(String(doc)).toBe('|\n\n')
+  })
```

Comment:
These are testing scalar string serialization, so don't belong in this block that's testing map serializations. Move these to somewhere near the end of the block starting on line 107.